Makefile

```diff
@@ -113,3 +114,14 @@ gendocs: docs/minikube.md
 docs/minikube.md: $(shell find cmd) $(shell find pkg/minikube/constants) pkg/minikube/cluster/assets.go
 	$(MKGOPATH)
 	cd $(GOPATH)/src/$(REPOPATH) && go run -ldflags="$(K8S_VERSION_LDFLAGS) $(MINIKUBE_LDFLAGS)" -tags gendocs gen_help_text.go
+
+.PHONY: deb
+deb: 
```

Comment:
Ah sure, the dependencies are target names that go after the :

So you could:

deb: out/minikube-linux-amd64

to depend on minikube-linux-amd64.